src/index.html

```diff
@@ -1,13 +1,75 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+
+  </head>
+
+  <body>
+
+    <div class="card" data-qa="card">
+
+      <img
+        class="picture"
```

Comment:
```suggestion
        class="card__picture"
```
Не нужно все элементы делать отдельными блоками. Делай блоком только то, что является отдельной логической единицей, которую можно переиспользовать потом. Картинку, строку с ценой или описание товара вряд ли можно назвать отдельным компонентом, который можно переиспользовать. Переделай их на элементы